lightning/src/ln/channelmanager.rs

```diff
@@ -6023,9 +6028,19 @@ where
 				InboundV1Channel::new(&self.fee_estimator, &self.entropy_source, &self.signer_provider,
 					counterparty_node_id.clone(), &self.channel_type_features(), &peer_state.latest_features,
 					&unaccepted_channel.open_channel_msg, user_channel_id, &self.default_configuration, best_block_height,
-					&self.logger, accept_0conf).map_err(|e| APIError::ChannelUnavailable { err: e.to_string() })
+					&self.logger, accept_0conf).map_err(|e| {
+						let err_str = e.to_string();
+						log_error!(self.logger, "{}", err_str);
```

Comment:
Since we are moving towards using `logger` wrapper, instead of `self.logger` to make logging robust.
It's better to incorporate it in this PR too.

```suggestion
						log_error!(logger, "{}", err_str);
```